python/core/auto_generated/symbology/qgsmarkersymbol.sip.in

```diff
@@ -245,6 +245,9 @@ rendered.
 
 If ``selected`` is ``True`` then the symbol will be drawn using the "selected feature"
 style and colors instead of the symbol's normal style.
+
+If ``generateMaskPainterPath`` is ``True`` then the symbol will generate a mask painter path and will
```

Comment:
This should be a flag in the render context flags instead of a new argument (adding arguments for things like this just makes the api awkward and complex)

Reply:
It's no longer needed, it's remains from an older implementation